elements/rh-navigation-secondary/rh-navigation-secondary-menu-section.ts

```diff
@@ -11,11 +11,11 @@ import styles from './rh-navigation-secondary-menu-section.css';
 /**
  * @summary 'A menu section which auto upgrades header and sibling link list accessibility attributes'
  *
- * @slot header     - Adds a header tag to section, expects `<h1>, <h2>, <h3>, <h4>, <h5>, <h6>`
- * @slot links      - Adds a ul tag to section, expects `<ul>, <ol>`
- * @slot cta        - Adds a section level CTA, expects `<rh-cta>`
+ * @slot header     - Adds a header tag to section, expects `<h1> | <h2> | <h3> | <h4> | <h5> | <h6>` element
+ * @slot links      - Adds a ul tag to section, expects `<ul> | <ol>` element
+ * @slot cta        - Adds a section level CTA, expects `<rh-cta>` element
  *
- * @csspart base    - container, <section> element
+ * @csspart container    - container, <section> element
```

Comment:
please add a patch changeset which indicates that the manifest css parts and properties entries were corrected

Reply:
> please add a patch changeset which indicates that the manifest css parts and properties entries were corrected

Thanks for pointing this out didn't think about that being a patch change but see now how it is.

Reply:
needs system token `@cssprop`s